SequenceSimulation/main.cc

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+#include <random>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <stdlib.h>
+#include "sequence.h"
+
+// This programm creates one multiple FASTA file with S random sequences of lenght L. 
+// If there is more then one sequence then we have mutations with mutation probability P. 
+// Call script with: ./main S L P
+// Example: ./main 5 100 0.3
+
+using namespace std;
+
+int main(int argc, char *argv[]){	
+    unsigned int numberOfSeqencens = atoi(argv[1]);
+    if (numberOfSeqencens <= 0) {
+        cerr << "\nThe first number must be an positive integer.\n" << endl;
+        printHelp();
+        return -1;
+    }
+    unsigned int DNAlength = atoi(argv[2]);
+    if (DNAlength <= 0) {
+        cerr << "\nThe second number must be an positive integer.\n"  << endl;
+        printHelp();
+        return -1;
+    }
+    double mutationpProbability = atof(argv[3]);
+    if (mutationpProbability < 0 || mutationpProbability > 1) {
+        cerr << "\nThe third number must be an double between 0 and 1.\n" << endl;
+        printHelp();
+        return -1;
+    }
+
+    vector<string> DNAsequences;
+    string DNA = randomDNA(DNAlength);
+    DNAsequences.push_back(DNA);
```

Comment:
This means that the first sequence in `DNAsequences` is an exception: On average it is more similar to sequences 2,3,... than they are to each other. This is unexpected and not documented in the code. It would even be relevant to the player.
In any case, I suggest you throw away the "mother of all sequences". This corresponds to having a star-like tree with the mother in the center and all extant sequences (from the present) are used as input for the alignment.